mne/decoding/transformer.py

```diff
@@ -79,10 +79,16 @@ def fit(self, epochs_data, y):
 
         for key, this_pick in picks_list.items():
             if self.with_mean:
-                ch_mean = X[:, this_pick, :].mean(axis=1)[:, None, :]
+                if len(this_pick) == 0:
+                    ch_mean = np.nan * np.ones((X.shape[0], 1, X.shape[2]))
+                else:
+                    ch_mean = X[:, this_pick, :].mean(axis=1, keepdims=True)
                 self.ch_mean_[key] = ch_mean  # TODO rename attribute
             if self.with_std:
-                ch_std = X[:, this_pick, :].mean(axis=1)[:, None, :]
```

Comment:
@kingjr I just noticed this error in `Scaler.fit`. This is bug (setting `self.ch_std_[key]` to the *mean*), right?